### DISCUSSION TITLE
Why does the multi-step and complex form examples not use `<Vueform>`?

### DISCUSSION THREAD

**Discussion by letmaik** [vueform/vueform#211] [Questions]
https://vueform.com/examples/basics-multi-step-form
https://vueform.com/examples/basics-complex-form

```vue
<template>
  <div class="bg-white rounded-lg p-10 max-w-lg shadow-box-circle dark:bg-dark-1000">
    <form class="w-full">
      <FormElements>
...
```

**Comment by adamberecz** [comment]
These examples intend to demonstrate the [`SFC`](https://vueform.com/docs/rendering-forms#using-single-file-component) rendering style. This can be useful when a form needs to be reusable, yet should accept props, and you can interact with its API, subscribe to its events directly, etc. as if it was a `<Vueform>` component.

**Reply by letmaik** [reply]
Why not create a separate example to demonstrate that? I feel like otherwise it's not clear what is being taught and it looks all a bit inconsistent and confusing.

**Reply by adamberecz** [reply]
The "Basics" examples were created first using a different approach than the others above and kinda left as it is. I agree though that is could/should be more consistent.
